datarobot_provider/example_dags/hospital_readmissions_xgboost_example.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2025 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+
+import datarobot as dr
+from airflow.decorators import dag
+
+from datarobot_provider.operators.ai_catalog import CreateDatasetFromRecipeOperator
+from datarobot_provider.operators.ai_catalog import CreateWranglingRecipeOperator
+from datarobot_provider.operators.ai_catalog import UploadDatasetOperator
+from datarobot_provider.operators.datarobot import CreateProjectOperator
+from datarobot_provider.operators.datarobot import GetOrCreateUseCaseOperator
+from datarobot_provider.operators.datarobot import GetProjectBlueprintsOperator
+from datarobot_provider.operators.datarobot import TrainModelsOperator
+from datarobot_provider.operators.model_registry import CreateRegisteredModelVersionOperator
+from datarobot_provider.operators.model_training import TrainModelOperator
+from datarobot_provider.sensors.datarobot import AutopilotCompleteSensor
+
+"""
+Example of Aiflow DAG for DataRobot data preparation and model training.
+Configurable parameters for this dag:
+* dataset_file_path - URL or a local path for a csv/parquet/xlsx file.
+* project_name - name of the project as displayed in DataRobot UI.
+* autopilot_settings - a dictionary with the modelling project autopilot settings.
+"""
+
+
+@dag(
+    schedule=None,
+    tags=["example", "csv", "wrangling", "modeling"],
+    params={
+        "dataset_file_path": "https://s3.amazonaws.com/datarobot_public_datasets/10k_diabetes.csv",
+        "project_name": "hospital-readmissions-example",
+        "autopilot_settings": {"target": "readmitted", "mode": "manual", "max_wait": 3600},
+    },
+)
+def hospital_readmissions_xgboost_example():
+    # Create a Use Case to keep all subsequent assets. Default name is "Airflow"
+    create_use_case = GetOrCreateUseCaseOperator(task_id="create_use_case", set_default=True)
+
+    # Upload the data into Data Registry.
+    upload_dataset = UploadDatasetOperator(task_id="upload_dataset")
+
+    # Define data preparation.
+    create_recipe = CreateWranglingRecipeOperator(
+        task_id="create_recipe",
+        dataset_id=upload_dataset.output,
+        dialect=dr.enums.DataWranglingDialect.SPARK,
+        # See the list of available *operation* options in the DtaRobot API documentation:
```

Comment:
```suggestion
        # See the list of available *operation* options in the DataRobot API documentation:
```

sorry for the nit - just noticed